Resources/Private/Fusion/Root.fusion

```diff
@@ -1,8 +1,31 @@
 // this is only included in dev context
+// todo not anymore
```

Comment:
thats a proplem as we cant use eel here.
either put some logic in `PropTypeExceptionHandler` to determine if in dev context and fall back to the general exception handler (fusion settings)
or we use context aware objects.yaml and use an interface which we set in `@exceptionHandler`

Reply:
we _could_ of course achieve the previous behaviour with aop and `\Neos\Neos\Domain\Service\FusionService::setPrependFusionIncludes`